README.md

```diff
@@ -1,13 +1,8 @@
-# Hermeto integration tests repository
+# A test for unusual location of Cargo.toml
 
-This repository hosts the integration tests for our project. Each branch represents a different test
-case. Please ignore README.md files in those branches at the moment. They might not be up to date.
-
-The branch names begin with a prefix indicating a package manager, followed by the
-test case name in the format: `<package-manager>/<test-case>`.
-
-To create a new test please create or request a new branch in GitHub UI first
-and then open a PR against this branch. Alternatively, you can push an empty
-branch first if you have push rights. Please avoid pushing non-empty branches
-since that would hinder the visibility of changes made to this repository in
-context of the main project repository.
+Current heuristic allows to find the root Cargo.toml.
+In most cases this heuristic could be simplified, but not
+always. This test guards against oversimplification of
+Cargo.toml search. It utilises specific version of cryptography
```

Comment:
I think we should be a bit more verbose on the explanation, something like:

_Most Python Rust extension packages come with a top-level Cargo.toml file, but not all. For those which do not our current heuristic is to look for the nearest Cargo.toml file reachable from the package's top-level directory.
This is a regression test to ensure we adhere to that heuristic and do not pick a Cargo.toml file from the list of found ones (e.g. in examples) arbitrarily. It utilizes specific...._